tools/c7n_gcp/c7n_gcp/resources/appengine.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2019 Capital One Services, LLC
```

Comment:
Unless your doing work under contract / employment paid for by capitalone, this isn't correct. This project uses contributor license agreements from apache foundation, which do not use copyright assignment. Hence these files should reflect the entity/organization of the author to be accurate.

Reply:
@jondavidi, @jonathana, @evhenyshaliov - could you advice whether it's right to keep it or not from the legal perspective?